Library/Formula/awscli.rb

```diff
@@ -100,7 +103,7 @@ def caveats; <<-EOS.undent
       #{HOMEBREW_PREFIX}/share/awscli/examples
 
     Add the following to ~/.bashrc to enable bash completion:
-      complete -C aws_completer aws
+      complete -C #{HOMEBREW_PREFIX}/etc/bash_completion.d/aws_completer aws
 
     Add the following to ~/.zshrc to enable zsh completion:
       source #{HOMEBREW_PREFIX}/share/zsh/site-functions/_aws
```

Comment:
I agree. But I was looking over other ports that call `bash_completion.install`, and something like 3 out of 40 (at time of writing) even allude to the existence of bash completion. We'd probably be more consistent if we removed the caveat entirely, but awscli is such a complex script, that completion is far more useful here than it is for, say, youtube-dl.

There's also no consistency on how the bash completion scripts are being named; I was considering tackling that in a separate PR.